### PR TITLE
[ODS-4881] Add SQL Server support to docker deployments

### PR DESCRIPTION
**Pull request by jamessmckay**

### CODE REVIEW

File: Compose/pgsql/compose-shared-instance-env-build.yml [hunk 1]
```diff
@@ -45,14 +45,14 @@ services:
     restart: always
     hostname: nginx
     volumes:
-      - ./Web-Gateway/ssl:/ssl/
+        - ../../ssl:/ssl/
```
Comment (tbmeador-dlp) [inline Compose/pgsql/compose-shared-instance-env-build.yml:48]:
```suggestion
      - ../../ssl:/ssl/
```

File: Web-Ods-AdminApp/Alpine/mssql/Dockerfile [hunk 2]
```diff
@@ -12,12 +12,16 @@ ENV PGBOUNCER_LISTEN_PORT=6432
...
+# Disable the globaliztion invariant mode (set in base image)
+ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false
+
 WORKDIR /app
 COPY appsettings.template.json /app/appsettings.template.json
 COPY run.sh /app/run.sh
 COPY log4net.config /app/log4net.txt
 
-RUN apk --no-cache add unzip=6.0-r8 dos2unix=7.4.2-r0 bash=5.1.0-r0 gettext=0.20.2-r2 postgresql-client=13.2-r0 jq=1.6-r1 && \
+RUN apk --no-cache add unzip=6.0-r8 dos2unix=7.4.2-r0 bash=5.1.0-r0 gettext=0.20.2-r2 postgresql-client=13.2-r0 jq=1.6-r1 icu-libs=56.1-r2 icu=67.1-r2 && \
```
Comment (tbmeador-dlp) [inline Web-Ods-AdminApp/Alpine/mssql/Dockerfile:24]: `icu-libs` is included here but not in the others, is that intentional?

File: Web-SwaggerUI/Alpine/Dockerfile [hunk 3]
```diff
@@ -13,12 +13,16 @@ ENV WebApiVersionUrl="https://localhost/${ODS_VIRTUAL_NAME}/"
...
+ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false
+
 WORKDIR /app
 COPY appsettings.template.json /app/appsettings.template.json
 COPY run.sh /app/run.sh
 
-RUN apk --no-cache add unzip=6.0-r8 dos2unix=7.4.2-r0 bash=5.1.0-r0 gettext=0.20.2-r2 && \
-    wget -O /app/SwaggerUI.zip https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_apis/packaging/feeds/EdFi/nuget/packages/EdFi.Suite3.Ods.SwaggerUI/versions/${VERSION}/content && \
+RUN apk --no-cache add unzip=6.0-r8 dos2unix=7.4.2-r0 bash=5.1.0-r0 gettext=0.20.2-r2 icu=67.1-r2 && \
+    wget -O /app/SwaggerUI.zip https://www.myget.org/F/ed-fi/api/v2/package/EdFi.Suite3.Ods.SwaggerUI/${VERSION} && \
```
Comment (tbmeador-dlp) [inline Web-SwaggerUI/Alpine/Dockerfile:25]: this needs to be reverted back to the azure feed

File: Web-Ods-Api/Alpine/pgsql/Dockerfile [hunk 4]
```diff
@@ -12,13 +12,17 @@ ENV PGBOUNCER_LISTEN_PORT=6432
...
+
 WORKDIR /app
 COPY appsettings.template.json /app/appsettings.template.json
 COPY run.sh /app/run.sh
 COPY log4net.config /app/log4net.txt
 
-RUN apk --no-cache add unzip=6.0-r8 dos2unix=7.4.2-r0 bash=5.1.0-r0 gettext=0.20.2-r2 postgresql-client=13.2-r0 && \
-    wget -O /app/WebApi.zip https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_apis/packaging/feeds/EdFi/nuget/packages/EdFi.Suite3.Ods.WebApi/versions/${VERSION}/content && \
+RUN apk --no-cache add unzip=6.0-r8 dos2unix=7.4.2-r0 bash=5.1.0-r0 gettext=0.20.2-r2 postgresql-client=13.2-r0 icu=67.1-r2 && \
+    wget -O /app/WebApi.zip  https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_apis/packaging/feeds/EdFi/nuget/packages/EdFi.Suite3.Ods.WebApi/versions/${VERSION}/content && \
```
Comment (tbmeador-dlp) [inline Web-Ods-Api/Alpine/pgsql/Dockerfile:25]:
```suggestion
    wget -O /app/WebApi.zip https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_apis/packaging/feeds/EdFi/nuget/packages/EdFi.Suite3.Ods.WebApi/versions/${VERSION}/content && \
```

File: Web-Ods-AdminApp/Alpine/mssql/Dockerfile (hunk 2 above)
Comment (jamessmckay) [reply Web-Ods-AdminApp/Alpine/mssql/Dockerfile:24]: this should only be icu not the icu-libs

File: Web-SwaggerUI/Alpine/Dockerfile (hunk 3 above)
Comment (jamessmckay) [reply Web-SwaggerUI/Alpine/Dockerfile:25]: done

**Comment by vimayya** [comment]
@jamessmckay main readme has links to compose files that have been moved.

File: Web-Gateway/generate-cert.sh [hunk 5]
```diff
@@ -1,18 +0,0 @@
-#!/bin/bash
```
Comment (tbmeador-dlp) [inline Web-Gateway/generate-cert.sh:1]: did this file not get moved to the new location?

Comment (jamessmckay) [reply Web-Gateway/generate-cert.sh]: done

**Review by tbmeador-dlp — APPROVED** [review]

**Pull request merged**